api/v2/openapi.yaml

```diff
@@ -0,0 +1,345 @@
+        - application/json
+      parameters:
+        - in: body
+          name: silence
+          description: the silence to create
+          required: true
+          schema:
+            $ref: '#/definitions/silence'
+      responses:
+        '200':
+          description: Created / updated silence response
+          schema:
+            type: object
+            properties:
+              silenceID:
+                type: string
+        '400':
+          description: Bad request
+          schema:
+            type: string
+  /silence/{silenceID}:
+    get:
+      tags:
+        - silence
+      operationId: getSilence
+      description: Get a silence by its id
+      parameters:
+        - in: path
+          name: silenceID
+          type: string
+          format: uuid
+          required: true
+          description: ID of the silence to get
+      responses:
+        '200':
+          description: Get silence response
+          schema:
+            $ref: '#/definitions/silence'
+        '404':
+          description: A silence with the specified id was not found
+        '500':
+          description: Internal server error
+          schema:
+            type: string
+    delete:
+      tags:
+        - silence
+      operationId: deleteSilence
+      description: Delete a silence by its id
+      parameters:
+        - in: path
+          name: silenceID
+          type: string
+          format: uuid
+          required: true
+          description: ID of the silence to get
+      responses:
+        '200':
+          description: Delete silence response
+        '500':
+          description: Internal server error
+          schema:
+            type: string
+  /alerts:
+    get:
+      tags:
+        - alert
+      operationId: getAlerts
+      description: Get a list of alerts
+      parameters:
+        - in: query
+          name: active
+          type: boolean
+          description: Show active alerts
+          default: true
+        - in: query
+          name: silenced
+          type: boolean
+          description: Show silenced alerts
+          default: true
+        - in: query
+          name: inhibited
+          type: boolean
+          description: Show inhibited alerts
+          default: true
+        - in: query
+          name: unprocessed
+          type: boolean
+          description: Show unprocessed alerts
+          default: true
+        - name: filter
+          in: query
+          description: A list of matchers to filter alerts by
+          required: false
+          type: array
+          items:
+            type: string
+        - name: receiver
+          in: query
+          description: A regex matching receivers to filter alerts by
+          required: false
+          type: string
+      responses:
+        '200':
+          description: Get alerts response
+          schema:
+            type: array
+            items:
+              '$ref': '#/definitions/alert'
+        '400':
```

Comment:
Maybe define references for default 400 and 500 error responses, since they are used in a few places?

```
responses:
  400:
    description: Bad request
    schema:
      type: string
  500:
    description: Internal server error
    schema:
      type: string
```